webapp/components/survey/SurveyForm/nodeDefs/components/types/nodeDefEntityFormGrid.js

```diff
@@ -40,44 +51,46 @@ const NodeDefEntityFormGrid = (props) => {
 
   const columns = NodeDefLayout.getColumnsNo(surveyCycleKey)(nodeDef)
   const rdgLayout = NodeDefLayout.getLayoutChildren(surveyCycleKey)(nodeDef)
-  const innerPageChildren = NodeDefLayout.rejectNodeDefsWithPage(surveyCycleKey)(childDefs)
-
-  return innerPageChildren.length > 0 ? (
-    <ResponsiveGridLayout
-      breakpoints={{ lg: 1200, md: 996, sm: 768, xs: 480, xxs: 0 }}
-      autoSize={false}
-      rowHeight={70}
-      cols={{ lg: columns, md: columns, sm: columns, xs: columns, xxs: 1 }}
-      layouts={{ lg: rdgLayout, md: rdgLayout, sm: rdgLayout, xs: rdgLayout }}
-      containerPadding={edit && canEditDef ? [15, 40] : [15, 15]}
-      margin={[5, 5]}
-      isDraggable={edit && canEditDef}
-      isResizable={edit && canEditDef}
-      compactType={null}
-      useCSSTransforms={true}
-      preventCollision={true}
-      className={mounted ? 'mounted' : ''}
-      onDragStop={onChangeLayout}
-      onResizeStop={onChangeLayout}
-    >
-      {innerPageChildren.map((childDef) => (
-        <div key={NodeDef.getUuid(childDef)} id={NodeDef.getUuid(childDef)}>
-          <NodeDefSwitch
-            edit={edit}
-            entry={entry}
-            preview={preview}
-            recordUuid={recordUuid}
-            surveyInfo={surveyInfo}
-            surveyCycleKey={surveyCycleKey}
-            nodeDef={childDef}
-            parentNode={node}
-            canEditDef={canEditDef}
-            canEditRecord={canEditRecord}
-            canAddNode={canAddNode}
-          />
-        </div>
-      ))}
-    </ResponsiveGridLayout>
+  const nodeDefsInnerPage = NodeDefLayout.rejectNodeDefsWithPage(surveyCycleKey)(childDefs)
+
+  return nodeDefsInnerPage.length > 0 ? (
+    <div className="survey-form__node-def-entity-form-grid-wrapper" data-child-def-names-ordered={childDefNamesOrdered}>
```

Comment:
can't this be data-child-defs ?

Reply:
data-child-names, because in the tests we would need the names, not the uuids of the defs